src/test/scala/com/atomist/rug/kind/elm/ElmNewStaticPageTest.scala

```diff
@@ -0,0 +1,77 @@
+package com.atomist.rug.kind.elm
```

Comment:
We need these tests right now, especially during the refactoring work I've been doing.